elisp/intero.el

```diff
@@ -1796,7 +1795,7 @@ type as arguments."
 (defun intero-format-get-type-at (beg end)
   "Compose a request for getting types in region from BEG to END."
   (format ":type-at %S %d %d %d %d %S"
-          (intero-localize-path (intero-temp-file-name))
+          (intero-path-for-ghci (intero-temp-file-name))
```

Comment:
I believe that the format string '%S' here should also be changed to '%s'.

Reply:
Indeed, thanks - this is now fixed.